CBLClient/Apps/CBLTestServer-Dotnet/TestServer/FileLoggingMehtod.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+using System.Net;
+using JetBrains.Annotations;
+using Couchbase.Lite.Logging;
+
+namespace Couchbase.Lite.Testing
+{
+    public class FileLoggingMehtod
+    {
+        public static void Configure([NotNull] NameValueCollection args,
+                                [NotNull] IReadOnlyDictionary<string, object> postBody,
+                                [NotNull] HttpListenerResponse response)
+        {
+            String log_level = postBody["log_level"].ToString();
+            String directory = postBody["directory"].ToString();
+            int max_rotate_count = (int)postBody["max_rotate_count"];
+            long max_size = (long)postBody["max_size"];
+            bool plain_text = Convert.ToBoolean(postBody["plain_text"]);
+
+            if (directory.Equals(""))
```

Comment:
c# provide IsNullOrEmpty method

Reply:
directory variable is a string. "" is not equal to null nor string object has isEmpty method, so can't use either.